### ISSUE TITLE
this honestly

### ISSUE THREAD

**Issue by Nexity** [o7-Fire/testa#55]
loss = 0.0
